keras_cv/models/backbones/resnet_v1/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022 The KerasCV Authors
```

Comment:
nit: 2023 (also in all the new resnet_v1_backbone files)

Reply:
Done.